File: Assets/Scripts/TimerGUI.js
static var timeElapsed:int;

function Start() {
	timeElapsed = 0;
}

function Update() {
	if (!BunnyCollect.allBunniesCaught) {
		timeElapsed = Time.time;
		guiText.text = "Time elapsed: " + timeFormat(timeElapsed) + " seconds";
	}
}

static function timeFormat(time: float) {
	var minutes:int = time / 60;
	var seconds:int = time % 60;
	return minutes + ":" + seconds;
}

File: bunnies-and-lozenges/Assets/CS_Scripts/TimerGUI.cs
// handles showing time elapsed during play

using UnityEngine;
using System.Collections;

public class TimerGUI : MonoBehaviour {

	public static int timeElapsed;

	void Start() {
		timeElapsed = 0;
	}
	
	void Update() {
		if (!BunnyCollect.allBunniesCaught) {
			timeElapsed = (int) Time.time;
			guiText.text = "Time elapsed: " + timeFormat(timeElapsed) + " seconds";
		}
	}
	
	public static string timeFormat(float time) {
		int minutes = (int)(time / 60);
		int seconds = (int)(time % 60);
		return minutes + ":" + seconds;
	}

}
